cmd/erasure-server-pool.go

```diff
@@ -33,7 +33,7 @@ import (
 
 	"github.com/dustin/go-humanize"
 	"github.com/google/uuid"
-	"github.com/minio/madmin-go/v2"
+	"github.com/minio/madmin-go"
```

Comment:
```suggestion
	"github.com/minio/madmin-go/v2"
```

Still needs to be /v2

Reply:
Yeah not sure why goimports sometimes messes this up.